JS-1-week-4/voice.js

```diff
@@ -0,0 +1,121 @@
+let userName = "";
+const toDo = [];
+const monthNames = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+const numericalRegex = /\d/;
+
+function getReply(command) {
+  // Handling name introduction
+  if (command.toLowerCase().startsWith("hello my name is ")) {
```

Comment:
It's a very good idea to clean up the user input with `toLowerCase`. You can also use `.trim()`.
Plus, to avoid repeating the operation all of the time, you can make a variable at the top of your function like this:
```js
function getReply(command) {
  const cleanCommand = command.toLowerCase().trim();

  if (cleanCommand.startWith("...") {
    ...
```

Reply:
Hi @vdaguenet ,Done the change .Please have a look.